src/NexusMods.App.UI/Pages/LibraryPage/LibraryViewModel.cs

```diff
@@ -224,19 +228,35 @@ public LibraryViewModel(
                 .Bind(out _collections)
                 .Subscribe()
                 .AddTo(disposables);
+
+            // Auto check updates on entering library.
+            RefreshUpdatesCommand.Execute(Unit.Default);
```

Comment:
What is the cache on this? I don't want us to send network requests every time the library is open

Reply:
There isn't at the moment.

I was considering adding a 1 minute one but given that I get razed for being overkill typically I didn't add one to the update service.

I don't consider the user rapidly switching between library and loadout on left menu to be a concern.

I do consider them mashing refresh to be a concern, but due to the above I've 'kept it simple'.

Reply:
> I don't consider the user rapidly switching between library and loadout on left menu to be a concern. No sane user would be doing that

All users will be doing that. We either add a 5 minute timeout or no automatic update.

Also this throws if you are logged out. Exception is captured at some point I think, but we shouldn't be trying.

Reply:
I will add a rate limit on the service in the upcoming PR.

Reply:
I've still plenty of bugs to fix around the functionality in general, I'll put this on my shortlist and handle it in follow up.